libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/component/map/ScalebarComponentTest.kt

```diff
@@ -0,0 +1,208 @@
+package com.mapbox.navigation.dropin.component.map
+
+import android.content.Context
+import android.view.Gravity
+import androidx.core.graphics.Insets
+import androidx.test.core.app.ApplicationProvider
+import com.mapbox.maps.MapView
+import com.mapbox.maps.plugin.scalebar.ScaleBarPlugin
+import com.mapbox.maps.plugin.scalebar.generated.ScaleBarSettings
+import com.mapbox.maps.plugin.scalebar.scalebar
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.dropin.MapboxMapScalebarParams
+import com.mapbox.navigation.testing.MainCoroutineRule
+import io.mockk.clearMocks
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import kotlin.random.Random
+
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class, ExperimentalCoroutinesApi::class)
+@RunWith(RobolectricTestRunner::class)
+internal class ScalebarComponentTest {
+
+    @get:Rule
+    var coroutineRule = MainCoroutineRule()
```

Comment:
```suggestion
    val coroutineRule = MainCoroutineRule()
```
The same in other tests.